main.rb

```diff
@@ -0,0 +1,10 @@
+require './library'
+
+library = Library.new
+library = File.exist?('./db/db.yml') ? library.load_dump : library.first_load
```

Comment:
`library.first_load` not works
check `Date.today`

Reply:
1. Can't reproduce problem with "#first_load"
2. Added require 'date'